packages/backend/test/prover.ts

```diff
@@ -0,0 +1,66 @@
+import path from 'path'
+import fs from 'fs'
+import * as snarkjs from 'snarkjs'
+import { SnarkProof, SnarkPublicSignals } from '@unirep/crypto'
+import { defaultProver } from '@unirep/circuits/provers/defaultProver'
+
+const buildPath = '../keys'
+
+/**
+ * The default prover that uses the circuits in default built folder `zksnarkBuild/`
+ */
+const _defaultProver = {
```

Comment:
We do we need another prover for the tests? Why don't we allow the backend prover to support unirep social proofs?

Reply:
I am not sure why `ava` cannot find the `defaultProver` from `@unirep-social/circuits`
It says
```
  Uncaught exception in test/airdrop.test.ts

  Error: Cannot find module '@unirep-social/circuits/provers/defaultProver'
```
Maybe we don't use `tsc` to build js in `@unirep-social/circuits`

or how can we use the ts `defaultProver`?